src/util-debug.c

```diff
@@ -1614,6 +1614,11 @@ void SCLogDeInitLogModule(void)
 #endif /* OS_WIN32 */
 }
 
+void SCFatalErrorOnInit(const char *arg)
```

Comment:
I think should have something is its name that indicates that it takes a simple string, no format string. Perhaps something like `SCFatalErrorOnInitStatic`.